tools/utils/parse-stack.ts

```diff
@@ -147,14 +177,14 @@ function parseStackFrames(frames) {
       return stop = true;
     }
 
-    if (_.isEmpty(ret)) {
+    if (!parsedFrames.length) {
```

Comment:
Hi @menewman! If you allow me, I'll like to make a suggestion. What do you think of the following change:

```suggestion
    if (parsedFrames.length === 0) {
``` 


I think this makes the code more explicit to follow.

Reply:
Just curious, how do you make a suggested change? I've never seen that feature before.

Reply:
Like this!

![Screen Shot 2020-04-16 at 3 04 17 PM](https://user-images.githubusercontent.com/747378/79511155-94e46d00-7ff3-11ea-90b3-4ae8c0108238.png)

Anyway, @MarceloPrado, I'm fine with that change. To me, checking for falsey length in JS is pretty idiomatic but I recognize everyone might not feel that way, and this certainly isn't something to bikeshed over.